next.config.mjs

```diff
@@ -5,12 +5,9 @@ const nextConfig = withYaml({
   poweredByHeader: false,
   reactStrictMode: true,
   swcMinify: true,
-  images: {
-    unoptimized: true,
-  },
   optimizeFonts: true,
-  output: 'export',
   experimental: {
+    bundlePagesExternals: true,
```

Comment:
just a fyi, if we update to next v15, we need to change this to `bundlePagesRouterDependencies` instead. just a note

Reply:
We also need to move this out of experimental in that case (it currently has no effect when testing but I think once we merge the contact pr it will) v15 is in canary still though and nextjs will tell us about the change when we update.